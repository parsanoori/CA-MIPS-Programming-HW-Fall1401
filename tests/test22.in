9
1003394886
199551765
416863872
635857043
410152494
976409510
884884531
127814155
769872272