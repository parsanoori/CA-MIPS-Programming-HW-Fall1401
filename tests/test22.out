127814155
199551765
410152494
416863872
635857043
769872272
884884531
976409510
1003394886